cloudnet-modules/cloudnet-cloudflare/src/main/java/de/dytanic/cloudnet/ext/cloudflare/CloudflareAPI.java

```diff
@@ -51,7 +51,7 @@ public static CloudflareAPI getInstance() {
         return CloudflareAPI.instance;
     }
 
-    public Pair<Integer, JsonDocument> createRecord(String serviceName, String email, String apiKey, String zoneId, DNSRecord dnsRecord) {
+    public Pair<Integer, JsonDocument> createRecord(String serviceName, String email, CloudflareConfigurationEntry.AuthenticationMethod authenticationMethod, String apiKey, String zoneId, DNSRecord dnsRecord) {
         Preconditions.checkNotNull(email);
```

Comment:
These check should only be done when using the AuthenticationMethod Global-Key, because the email is not needed when using the Bearer Token (could be set to null, although an empty string is working fine)

Reply:
Just make them authentication-method sensitive (authenticationMethod == BEARER || email != null)